Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.13.0-slim
```

Comment:
The specified Python version `3.13.0` does not exist. Consider using a stable version like `3.11.0-slim` to ensure compatibility and stability.